File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraTabbedMdi;
using DevExpress.Utils.Controls;
using DevExpress.Utils;

namespace WindowsApplication1
{
    public partial class Form1 : Form
    {
        Random r = new Random();
        Color GetRandomColor()
        {
            return Color.FromArgb(r.Next(255), r.Next(255), r.Next(255));
        }

        public Form1()
        {
            InitializeComponent();
            for (int i = 0; i < 30; i++)
            {
                Form form = new Form();
                form.MdiParent = this;
                form.Show();
                form.Text = String.Format("Page{0}", i);
                form.BackColor = GetRandomColor();
            }
        }
    }
}

File: VB/WindowsApplication1/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraTabbedMdi
Imports DevExpress.Utils.Controls
Imports DevExpress.Utils

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form
		Private r As New Random()
		Private Function GetRandomColor() As Color
			Return Color.FromArgb(r.Next(255), r.Next(255), r.Next(255))
		End Function

		Public Sub New()
			InitializeComponent()
			For i As Integer = 0 To 29
				Dim form As New Form()
				form.MdiParent = Me
				form.Show()
				form.Text = String.Format("Page{0}", i)
				form.BackColor = GetRandomColor()
			Next i
		End Sub
	End Class
End Namespace